docs/sphinx/index.rst

```diff
@@ -1,101 +1,86 @@
-   $ git clone https://github.com/llnl/raja.git 
-
-Then, you can build RAJA like any other CMake project, provided you have a C++
-compiler that supports the C++11 standard. The simplest way to build the code
-is to do the following in the top-level RAJA directory (in-source builds are 
-not allowed!):
-
-.. code-block:: sh
-
-    $ mkdir build
-    $ cd build
-    $ cmake ../
-    $ make
-
-For details about RAJA configuration and build options, please see 
-:doc:`config_build`.
-
-If you are interested in keeping up with RAJA development and communicating 
-with developers and users, please join our mailing list over at Google Groups -
-`RAJA Google Group <https://groups.google.com/forum/#!forum/raja-users>`_.
+many LLNL applications. RAJA is rooted in a perspective based on decades
+of experience working on production mesh-based multiphysics applications at 
+LLNL. Another goal of RAJA is to enable application developers to specialize 
+RAJA concepts for different code implementation patterns and C++ usage, 
+since data structures and algorithms vary widely across applications.
+
+RAJA helps developers insulate application loops from underlying 
+architecture and programming model-specific implementation details. Loop 
+bodies and traversals are decoupled via C++ lambda functions (loop bodies) 
+and C++ templates (traversal methods). This approach promotes tuning loop 
+patterns rather than individual loops, and makes it straightforward to
+parametrize loop execution policies so an application can be compiled in 
+a specific configuration suitable to a given architecture. 
+
+If you are new to RAJA, check out our :ref:`getting_started-label` Guide. It
+contains basic information about accessing the RAJA code, building it, and
+RAJA usage.
+
+See the :doc:`features` for detailed documentation for each feature.
+
+If you are interested in keeping up with RAJA development and communicating
+with developers and users, please join our `Google Group
+<https://groups.google.com/forum/#!forum/raja-users>`_.
 
 If you have questions, find a bug, or have ideas about expanding the
 functionality or applicability of RAJA and are interested in contributing
 to its development, please do not hesitate to contact us. We are always
 interested in improving RAJA and exploring new ways to use it.
 
-.. seealso::
-
-   More detailed discussion of RAJA, including its use in proxy apps included
-   in the RAJA source distribution can be found in these reports:
-
-   `The RAJA Portability Layer: Overview and Status (2014) <_static/RAJAStatus-09.2014_LLNL-TR-661403.pdf>`_
-
-   `RAJA Overview (extracted from ASC Tri-lab Co-design Level 2 Milestone Report 2015) <_static/RAJAOverview-Trilab-09.2015_LLNL-TR-677453.pdf>`_
-
-   Newer reports and other material will appear here over time.
-
-
-**Contents:**
-
 .. toctree::
    :maxdepth: 2
 
-   primer
-   basic_examples
-   advanced_examples
-   config_build
-   dev_guide
-   future
-   raja_license
+   getting_started
```

Comment:
@rhornung67  - did you remove the sectioning on this TOC? I thought it was good to have the sections subdivided in this list.

Reply:
@davidbeckingsale I removed them because they were not producing understandable toc links in the left column; e.g., when you go down in the page hierarchy beyond a certain depth there was no way to get back out to the top-level without reloading the docs from scratch.

We can try adding this stuff back in if you want after I get the section delimiters and cross-references fixed. I should be able to get this done after lunch.